apkid/rules/elf/packers.yara

```diff
@@ -289,3 +289,28 @@ rule upx_unknown_version_unmodified : packer
     upx_unmodified and
     not upx_compressed_apk
 }
+
+rule ollvm_obfuscator : obfuscator
+{
+  meta:
+    description = "Obfuscator-LLVM native"
+
+  strings:
+    $clang_version = "Obfuscator-LLVM clang version"
+    $based_on = "(based on Obfuscator-LLVM"
+
+  condition:
+    all of them
+}
+
+rule arxan_native : obfuscator
+{
+  meta:
+    description = "Arxan native"
+
+  strings:
+    $jni_export = "Java_xxxxxx_"
```

Comment:
Is there anything else that could be used here? Perhaps opcodes from a decryption method or something?

Reply:
I never saw it before. I have absolutely no idea how to fingerprint this.